src/RageSoundReader_Resample_Good.cpp

```diff
@@ -120,8 +120,6 @@ namespace
 			i2 = i1;
 			i1 = iRem;
 		}
-
-		return i1;
```

Comment:
Let's just change the entire body of this function to `return std::gcd(i1, i2);` (need to `#include<numeric>`)